src/resources/css/rsvp.css

```diff
@@ -187,4 +187,18 @@
 .event-tickets-meta-label {
 	font-weight:700;
 	margin: 0 1em 0 0;
-}
\ No newline at end of file
+}
+
+/* TwentySixteen fix for box shadow as bottom border on <a> link */
+
+.entry-content a,
+.entry-summary a,
+.taxonomy-description a,
+.logged-in-as a,
+.comment-content a,
+.pingback .comment-body > a,
+.textwidget a,
+.entry-footer a:hover,
+.site-info a:hover {
+	box-shadow: none;
+}
```

Comment:
I'm slightly torn on this one, while it's unlikely that we'll see this anywhere else (for a while at least) I was honestly surprised at it's usage here.  I decided to target all themes on the off chance that there are some 'copycats' out there...this particular style is super ugly on a few of our links.  Input is welcome :)